presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -75,6 +75,7 @@
     private Duration queryMaxCpuTime = new Duration(1_000_000_000, TimeUnit.DAYS);
 
     private DataSize queryMaxScanRawInputBytes = DataSize.succinctDataSize(1000, PETABYTE);
+    private long queryMaxOutputPositions = 1_000_000_000;
```

Comment:
`Long.MAX_VALUE`? Otherwise, it could regression in prod queries.

Reply:
Good point. Done. Thanks